bundles/redhat8.8/packages.txt.gotmpl

```diff
@@ -36,3 +36,5 @@ kernel-devel
 gssproxy
 libverto-module-base
 libverto
+kernel-headers
+kernel-devel
```

Comment:
These are already set on lines 34 and 35

Reply:
:man_facepalming: nice catch thank you